docs/tools-and-sdks.mdx

```diff
@@ -19,6 +19,10 @@ Allows other Stellar apps to request transaction signing or identity verificatio
 
 View all anchors on Stellar, their currencies, and where they operate.
 
+### [Asset Sandbox](https://stellar.cheesecakelabs.com/?utm_source=stellar&utm_medium=landing-page&utm_campaign=stellar-asset-sandbox)
```

Comment:
I be the answer is "yes," but feel like I should ask: Do we want the `?utm_source=stell....` included in the link?

Reply:
I'm not sure? I just copied the link from the blog https://stellar.org/blog/app-for-issuing-tokens-assets-on-stellar-testnet